sale_triple_discount/models/sale_order.py

```diff
@@ -15,3 +16,35 @@ def _amount_all(self):
             prev_values.update(order.order_line.triple_discount_preprocess())
         super(SaleOrder, self)._amount_all()
         self.env['sale.order.line'].triple_discount_postprocess(prev_values)
+
+    @api.multi
+    def _get_tax_amount_by_group(self):
+        # Copy/paste from standard method in sale
+        self.ensure_one()
+        res = {}
+        for line in self.order_line:
+            price_reduce = line.price_reduce  # changed
+            taxes = line.tax_id.compute_all(
+                price_reduce, quantity=line.product_uom_qty,
+                product=line.product_id,
+                partner=self.partner_shipping_id)['taxes']
+            for tax in line.tax_id:
```

Comment:
It seems tests do not enter this loop. Could you see ?

Reply:
As commented this is a copy/paste from the standard method with the only change definitely matching what this module must do, and it's explicitly commented. So IMO not having it covered should not be blocking...

Reply:
@grindtildeath I don't block but just focus attention to avoid future regression as this module already caused some problems.